ibis/impala/compiler.py

```diff
@@ -181,6 +185,111 @@ def _cumulative_to_window(translator, expr, window):
     return new_expr
 
 
+_window_types = {
+    'row': 'ROWS',
```

Comment:
If we use `'rows'` instead of `'row'` then we can get rid of the `_window_types` variable.